src/main/java/fr/insalyon/creatis/moteur/plugins/workflowsdb/PluginConfiguration.java

```diff
@@ -51,7 +51,7 @@ public class PluginConfiguration {
     private String driverClass;
     private String url;
 
-    public static PluginConfiguration getInstance() throws Exception {
+    public static PluginConfiguration getInstance() throws WorkflowsDBException {
```

Comment:
can you see the pmd warnings in this file ?

Reply:
what ? I have changed that before doing the PMD because I thought It was cleaner.